doc/src/manual/faq.md

```diff
@@ -552,7 +552,7 @@ the loop, but it cannot algebraically reduce multiple operations into fewer equi
 
 The most reasonable alternative to having integer arithmetic silently overflow is to do checked
 arithmetic everywhere, raising errors when adds, subtracts, and multiplies overflow, producing
-values that are not value-correct. In this [blog post](http://danluu.com/integer-overflow/), Dan
+values that are not value-correct. In this [blog post](https://danluu.com/integer-overflow/), Dan
```

Comment:
We should make this change. @EternalLearner42 do you mind rebasing with this change? (Or opening a new PR addressing this one) I think most of the others have already been updated on master, but please let me know if not. Thanks again for the work. Sorry this looks like it just got forgotton.

Reply:
It did indeed get forgotten, amidst a pile of college work. Since I pivoted to another account and only now remembered this one, I didn't even get a notification for this. Deepest apologies for that.

Reply:
No worries I already took care of this.